catroid/src/androidTest/java/org/catrobat/catroid/test/ui/BaseExceptionHandlerTest.java

```diff
@@ -46,14 +44,10 @@
 	@Before
 	public void setUp() throws Exception {
 		super.setUp();
-		context = InstrumentationRegistry.getTargetContext();
-		sharedPreferences = PreferenceManager.getDefaultSharedPreferences(context);
-		editor = sharedPreferences.edit();
-		editor.clear();
-		editor.putBoolean(SettingsActivity.SETTINGS_CRASH_REPORTS, true);
-		editor.commit();
-		CrashReporter.setIsCrashReportEnabled(true);
-		CrashReporter.initialize(context);
+		CrashReporterTest.setUpMethod();
+		context = CrashReporterTest.findContext();
+		sharedPreferences = CrashReporterTest.getSharedPreferences();
+		editor = CrashReporterTest.getEditor();
```

Comment:
> However, I'm not sure if it's worth it. What do you think?

Yes, you are right. Creating a method just for enabling and initializing crash reporting isn't worth it.